### PR TITLE
en, zh: add ebs-csi-node toleration

### LINKED ISSUE #1479: Improve EKS CSI doc to avoid user problems about toleration
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

Now users may fail to deploy tidbcluster in EKS when they deploy CSI with default config. Some CSI nodes can't tolerate our custom taint

2. Describe your suggestion or addition.

Add a notation about this problem

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by better0332**
### What is changed, added, or deleted? (Required)

add ebs-csi-node toleration

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [*] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/issues/1479
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [*] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

fix #1479

### CODE REVIEW

File: en/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -195,7 +195,13 @@ The following example shows how to create and configure a StorageClass for the `
 
 1. Deploy the [AWS EBS Container Storage Interface (CSI) driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) on the EKS cluster. If you are using a storage type other than `gp3`, skip this step.
 
-2. Create a `StorageClass` resource. In the resource definition, specify your desired storage type in the `parameters.type` field.
+2. Set ebs-csi-node `toleration`
+    ```bash
+    kubectl patch -n kube-system ds ebs-csi-node -p '{"spec":{"template":{"spec":{"tolerations":[{"operator":"Exists"}]}}}}'
+    daemonset.apps/ebs-csi-node patched
```
Comment (csuzhangxc) [inline en/deploy-on-aws-eks.md:201]: 要区分一下命令和它的输出吗

**Review by csuzhangxc — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Comment by ran-huang** [comment]
@better0332 Does the change apply to v1.2?

**Comment by better0332** [comment]
> @better0332 Does the change apply to v1.2?

@ran-huang yes, I have checked

File: en/deploy-on-aws-eks.md [hunk 2]
```diff
@@ -195,7 +195,21 @@ The following example shows how to create and configure a StorageClass for the `
...
-2. Create a `StorageClass` resource. In the resource definition, specify your desired storage type in the `parameters.type` field.
+2. Set ebs-csi-node `toleration`.
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    kubectl patch -n kube-system ds ebs-csi-node -p '{"spec":{"template":{"spec":{"tolerations":[{"operator":"Exists"}]}}}}'
+    ```
+
+   Expected output:
```
Comment (ran-huang) [inline en/deploy-on-aws-eks.md:206]:
```suggestion
    Expected output:
```

**Pull request merged**